graviti/portex/builtin.py

```diff
@@ -56,6 +56,15 @@ def __init__(self, **kwargs: Any) -> None:
     def _from_pyarrow(cls: Type[_T], pyarrow_type: pa.DataType) -> _T:
         return cls()
 
+    def to_builtin(self: _T) -> _T:
```

Comment:
do we need to define `to_builtin` method in `PortexType`?

Reply:
The return type of `to_builtin` is `PortexBuiltinType`. I don't think it is appropriate to define this in the parent class.